CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contribute to Puroto Repository
+
+Thank you for taking a moment to consider contributing to Puroto! You're awesome.
+
+> Before contributing, please read our [Code of Conduct](https://github.com/PurotoApp/.github/CODE_OF_CONDUCT.md) first. We want this community to stay civil and respectful to each other.
+
+Contributing can be done in many ways, also in more than 'just' code. Available options you can consider are:
+
+- Implement a feature or fix a bug
+- Create issues on bugs and feature-requests
+- Fund the development
+- Spread the word!
```

Comment:
We might also want to add some kind of information that all projects are open for contributions and that people can contribute code in GO or TS.

Reply:
This is fairly easy to guess when looking on the languages used by the orgs and the repo ig?

Reply:
Moved to #4.